OpenUtau/Classic/VoicebankInstaller.cs

```diff
@@ -96,6 +96,8 @@ class FileLoc {
                             while (!streamReader.EndOfStream) {
                                 string line = streamReader.ReadLine().Trim();
                                 var s = line.Split(new char[] { '=' }, StringSplitOptions.RemoveEmptyEntries);
+                                if (s.Length != 2)
+                                    s = line.Split(new string[] { ": " }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Thanks for the PR! Could you make this split using char ':', then call string.Trim() on each part? Otherwise it looks good to me.

Reply:
Ok, now the parts are trimmed after the line is split.